Blish HUD/GameServices/ModuleService.cs

```diff
@@ -317,20 +320,61 @@ public class ModuleService : GameService {
         }
 
         private void Gw2Instance_Gw2Started(object sender, EventArgs e) {
-            foreach (var module in _modules) {
-                if (module.State.Enabled) {
-                    module.TryEnable();
+            var resolvedDependencyOrder = SortByDependency(_modules, module => {
+                var dependencyModules = _modules.Where(m => {
+                    // Get all modules which have a dependency on the current module
+                    return module.Manifest?.Dependencies?.Any(dm => !dm.IsBlishHud && dm.Namespace == m.Manifest.Namespace) ?? false;
+                });
+
+                return dependencyModules;
+            }).ToList();
+
+            if (resolvedDependencyOrder != null) {
```

Comment:
In what scenario could resolvedDependencyOrder be null?

Reply:
I have no idea. SonarCloud wanted it there

Reply:
Removed.